stubs/SQLAlchemy/sqlalchemy/sql/operators.pyi

```diff
@@ -7,13 +7,13 @@ _T = TypeVar("_T")
 div = truediv
 
 class Operators:
-    def __and__(self, other): ...
-    def __or__(self, other): ...
-    def __invert__(self): ...
-    def op(self, opstring, precedence: int = ..., is_comparison: bool = ..., return_type: Any | None = ...): ...
-    def bool_op(self, opstring, precedence: int = ...): ...
-    def operate(self, op, *other, **kwargs) -> None: ...
-    def reverse_operate(self, op, other, **kwargs) -> None: ...
+    def __and__(self, other) -> Any: ...
+    def __or__(self, other) -> Any: ...
+    def __invert__(self) -> Any: ...
+    def op(self, opstring, precedence: int = ..., is_comparison: bool = ..., return_type: Any | None = ...) -> Any: ...
+    def bool_op(self, opstring, precedence: int = ...) -> Any: ...
+    def operate(self, op, *other, **kwargs) -> Any: ...
+    def reverse_operate(self, op, other, **kwargs) -> Any: ...
```

Comment:
Please don't set the return type to `Any`, except if we know that there is no other way to type these return types. Just leave out the return type annotation.